### DISCUSSION TITLE
该课程需要对TVM的掌握程度

### DISCUSSION THREAD

**Discussion by hbsun2113** [mlc-ai/mlc-zh#18] [Q&A]
授课过程中使用了TVM，但我本人对TVM完全不熟悉，希望老师在用到TVM的时候能先解释下该操作的作用。
如果该课程默认需要对TVM需要有一定掌握，请问需要掌握到什么程度。如果需要提前自学，请问有什么推荐资料，希望能附上链接，谢谢！

**Comment by Hzfengsy** [comment]
感谢你的提问。本课程主要介绍的是MLC本身的技术，以TVM为例进行展开。课程不要求对TVM有提前对掌握程度，课程过程中会在一定程度上介绍TVM的用法（如在Lecture 2中介绍了如何安装、如何构建module）。

**Reply by yjmm10** [reply]
请教一下，mlc是开源的还是可以先免费试用呢

**Reply by Hzfengsy** [reply]
mlc课程的材料都在这个仓库内，基于Apache 2.0协议开源
